src/app/qml/main.qml

```diff
@@ -288,14 +292,16 @@ ApplicationWindow {
         } else if (mainLayout.state == "drivePage") {
             if (selectedOption == Units.MainSelect.Write || downloadManager.isDownloaded(releases.selected.version.variant.url))
                 return qsTr("Write")
+            if (canOnlyDownloadFile) 
+                return qsTr("Download")
             if (Qt.platform.os === "windows" || Qt.platform.os === "osx") 
                 return qsTr("Download && Write")
             return qsTr("Download & Write") 
         } else if (mainLayout.state == "downloadPage") {
             if (releases.variant.status === Units.DownloadStatus.Write_Verifying || releases.variant.status === Units.DownloadStatus.Writing || releases.variant.status === Units.DownloadStatus.Downloading || releases.variant.status === Units.DownloadStatus.Download_Verifying)
                 return qsTr("Cancel")
             else
-                return qsTr("Retry")
```

Comment:
We will still need option to `Retry` when for example an USB drive is unplugged during the download process.